Presentation.Web/Controllers/API/V1/OrganizationRegistrationController.cs

```diff
@@ -0,0 +1,136 @@
+﻿using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Web.Http;
+using Core.ApplicationServices.Model.Organizations;
+using Core.ApplicationServices.Organizations;
+using Core.DomainModel.ItContract;
+using Core.DomainModel.Organization;
+using Core.DomainServices;
+using Presentation.Web.Controllers.API.V1.Mapping;
+using Presentation.Web.Infrastructure.Attributes;
+using Presentation.Web.Models.API.V1;
+using Presentation.Web.Models.API.V1.Organizations;
+using Swashbuckle.Swagger.Annotations;
+
+namespace Presentation.Web.Controllers.API.V1
+{
+    [PublicApi]
+    [RoutePrefix("api/v1/organization-registrations")]
+
+    public class OrganizationRegistrationController: BaseApiController
```

Comment:
Is it not about organization**unit** registrations?

Why not think more about naming of the route e.g. 

`api/v1/organizations/{orgUuid}/organization-units/{orgUnitUuid}` as base route and then

`api/v1/organizations/{orgUuid}/organization-units/{orgUnitUuid}/registrations` for all endpoints the GET or DELETE concrete registrations..

The new endpoint for DELETE should just be on `api/v1/organizations/{orgUuid}/organization-units/{orgUnitUuid}`

And then obsolete the existing delete endpoint in the other controller